src/guide/6-agent-birth-death/2-agent-birth-host.rst

```diff
@@ -1,3 +1,17 @@
 Agent Birth from Host Functions
 ===============================
 
+Agents can be created using host step functions. 
+
+.. tabs::
+  
+  .. code-tab:: cpp
+
+    FLAMEGPU_STEP_FUNCTION(BasicOutput) {
+        // Get the agent type 'agent'
+        auto t = FLAMEGPU->agent("agent");
+
+        // Create NEW_AGENT_COUNT new 'agent' agents with 'x' set to 1.0f
+        for (unsigned int i = 0; i < NEW_AGENT_COUNT; ++i)
+            t.newAgent().setVariable<float>("x", 1.0f);
+    }
```

Comment:
You can also do agent birth with `DeviceAgentVector`, however that's discouraged outside of init functions due to it being expensive.